azkaban-common/src/test/java/azkaban/utils/TestUtils.java

```diff
@@ -77,4 +81,19 @@ public static UserManager createTestXmlUserManager() {
     return manager;
   }
 
+  /**
+   * Reads a resource into a String
+   *
+   * @param name Relative path to the resource (relative to the parent object's package)
+   * @param parent Instance of the class to use in finding the resource. The resource is looked up in the same package
+   * where the class of the parent object is in.
+   * @return Resource content as a String
+   * @throws IOException if an I/O error occurs
+   */
```

Comment:
Added this javadoc ^

Is it ok?

Reply:
LGTM. @HappyRay WDYT?

Reply:
It's not my preferred choice. But I am ok with it.